spec/segment_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+
+describe I18n::JS::Segment do
+
+  let(:file)        { "tmp/i18n-js/segment.js" }
+  let(:translations){ { "en" => { "test" => "Test" }, "fr" => { "test" => "Test2" } } }
+  let(:options)     { {} }
+  subject { I18n::JS::Segment.new(file, translations, options) }
+
+  describe ".new" do
+
+    it "should persist the file path variable" do
```

Comment:
I rather use the first one, otherwise setting `subject` is kind of pointless.
And it also reads better!